cermine-impl/src/main/java/pl/edu/icm/cermine/CommandLineOptionsParser.java

```diff
@@ -72,9 +75,43 @@ public String parse(String[] args) throws ParseException {
         return null;
     }
 
-    public String getPath() {
+    public String getPath () {
         return commandLine.getOptionValue("path");
     }
+
+    public String getOutPath () {
+        if (!commandLine.hasOption("outpath")) {
+            return commandLine.getOptionValue("path");
+        } else {
+            return commandLine.getOptionValue("outpath");
+        }
+    }
+
+    public Long getLimit() {
```

Comment:
Why are you using Long instead of int for this option? The same question applies to the number of workers.